docs/api/index.rst

```diff
@@ -37,8 +37,8 @@ For those wishing to add custom functionality to their project.
 
 
 .. autosummary::
-   :hidden:
```

Comment:
This worked in part, but was generating warnings. I think there is a better solution but I'm still looking for it.